tests/test_python.py

```diff
@@ -140,6 +155,10 @@ def test_dso_name_in_pyperf_profile(
     insert_dso_name: bool,
     profiler_state: ProfilerState,
 ) -> None:
+    # pyperf is not working on aarch right now https://github.com/Granulate/gprofiler/issues/499
+    if is_aarch64() and profiler_type == "pyperf":
+        pytest.skip("PyPerf doesn't support aarch64 architecture!")
```

Comment:
```suggestion
        pytest.skip("PyPerf doesn't support aarch64 architecture, see https://github.com/Granulate/gprofiler/issues/499")
```
and can remove the comment before